### ISSUE TITLE
After Tabbing to a Link, Reading Forward skips text 

### ISSUE THREAD

**Issue by Githubuser4329** [nvaccess/nvda#12905]
### Steps to reproduce: 
1) tab to a hyperlink on the page. 
2) use arrow key to read forward 
- the next piece of text should be read off but NVDA often skips part of it and begins reading off halfway through the content beside the link. 

### Actual behavior:
NVDA skips reading the text that comes after a hyperlink. 

### Expected behavior: 
The text after a hyperlink should be read off when using arrow key for reading order 

#### NVDA installed/portable/running from source:
Installed 

#### NVDA version:
2021.2 

#### Windows version:
Windows10 

#### Name and version of other software in use when reproducing the issue:
Chrome (latest version) 

#### Other information about your system:
Dell computer 

#### Does the issue still occur after restarting your computer?
Yes 
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, this issue also happens on 2021.1 and 2020.4

**Comment by ehollig** [comment]
Hello @Githubuser4329, could you provide an example of where this is occuring? Also, would you happen to have Use screen layout (when supported) in the settings for browse mode checked? If so, NVDA presents information on screen, as it does visually. If you tab to a link, and press down arrow, NVDA will skip text that is on the same visual line as the link and jump to the next line of text.

**Comment by feerrenrut** [comment]
We are unable to process this issue as it stands.

- To be able to reproduce this issue, we need a code sample. Provide a minimal HTML sample to reproduce this using [codepen](https://codepen.io/).
